packages/grafana-ui/src/saga-themes/code/themes/dark.ts

```diff
@@ -0,0 +1,67 @@
+export default {
+  color: {
+    background: {
+      ui: {
+        canvas: 'hsl(220 11.8% 7.2%)',
+        primary: 'hsl(220 11.8% 10.8%)',
+        secondary: 'hsl(220 11.8% 15.6%)',
+        tertiary: 'hsl(220 11.8% 28.8%)',
+        overlay: 'hsl(220 11.8% 10.8% / 0.4)',
+      },
+      system: {
+        success: 'hsl(149 66% 5.4%)',
+        warning: 'hsl(36 100% 9%)',
+        error: 'hsl(336 87.4% 7.87%)',
+        info: 'hsl(220 67.2% 9.81%)',
+      },
```

Comment:
would have no idea how to use these, and what would we use for hover colors? (button hover state for example), and for transparent (minor tint backgrounds). 

Seem to be missing a lot of tokens, and now spread all over